src/fury.js

```diff
@@ -43,7 +43,7 @@ class Fury {
    * then uses the adapter to convert into refract elements and loads
    * these into objects.
    */
-  parse({source, mediaType, generateSourceMap = false}, done) {
+  parse({source, mediaType, generateSourceMap = false, parserOptions}, done) {
```

Comment:
As a user of the fury.js library, you can provide the `generateSourceMap` option in the `adapterOptions`. We can leave the other explicit `generateSourceMap` option in the existing place for backwards compatibility until we release fury 3.